formats/completed_transaction/frontend/examples/completed_transaction_with_promotion.json

```diff
@@ -0,0 +1,52 @@
+{
+  "details": {
+    "external_related_links": [
+
+    ],
+    "promotion": {
+      "url": "http://www.register-to-vote.gov.uk",
+      "choice": "register_to_vote"
```

Comment:
We've gone for `category` to avoid confusion with `type` which already exists in the schema ⚡️